answer/templates.html

```diff
@@ -17,6 +17,10 @@
 </script>
 <!-- runtimeFilters -->
 <script type="text/template" id="runtimeFilters">
-     runtimeFilters: [],
+     runtimeFilters: [{
+          columnName: 'color',
```

Comment:
Can you change this to `columName1`? and value red to `value1`. Do the same with all the other occurences. You can maybe add `// eg: color` at the line end instead